tests/more_string_manipulation/test_challenge_083.py

```diff
@@ -0,0 +1,13 @@
+# MORE STRING MANIPULATION CHAPTER
+
+# Challenge 083
+
+from exercises.more_string_manipulation import challenge_083
+
+
+def test_check_if_uppercase():
+    message = "GREAT"
```

Comment:
Parameterise this test to take more examples at the very least you need to test the false branch